src/engine/bank08.asm

```diff
@@ -6,44 +6,44 @@ unknown_data_20000: MACRO
 ENDM
 
 Data_20000: ; 20000 (8:4000)
-	unknown_data_20000 $07, POTION,                 CheckIfPotionPreventsKnockOut, AIPlayPotion
-	unknown_data_20000 $0a, POTION,                 FindTargetCardForPotion, AIPlayPotion
-	unknown_data_20000 $08, SUPER_POTION,           CheckIfSuperPotionPreventsKnockOut, AIPlaySuperPotion
-	unknown_data_20000 $0b, SUPER_POTION,           FindTargetCardForSuperPotion, AIPlaySuperPotion
-	unknown_data_20000 $0d, DEFENDER,               CheckIfDefenderPreventsKnockOut, AIPlayDefender
-	unknown_data_20000 $0e, DEFENDER,               CheckIfDefenderPreventsRecoilKnockOut, AIPlayDefender
-	unknown_data_20000 $0d, PLUSPOWER,              $4501, AIPlayPluspower
-	unknown_data_20000 $0e, PLUSPOWER,              $45a5, AIPlayPluspower
-	unknown_data_20000 $09, SWITCH,                 $462e, $4612
-	unknown_data_20000 $07, GUST_OF_WIND,           $467e, $4666
-	unknown_data_20000 $0a, GUST_OF_WIND,           $467e, $4666
-	unknown_data_20000 $04, BILL,                   $4878, $486d
-	unknown_data_20000 $05, ENERGY_REMOVAL,         $4895, $4880
-	unknown_data_20000 $05, SUPER_ENERGY_REMOVAL,   $49bc, $4994
-	unknown_data_20000 $07, POKEMON_BREEDER,        $4b1b, $4b06
-	unknown_data_20000 $0f, PROFESSOR_OAK,          $4cc1, $4cae
-	unknown_data_20000 $0a, ENERGY_RETRIEVAL,       $4e6e, $4e44
-	unknown_data_20000 $0b, SUPER_ENERGY_RETRIEVAL, $4fc1, $4f80
-	unknown_data_20000 $06, POKEMON_CENTER,         $50eb, $50e0
-	unknown_data_20000 $07, IMPOSTER_PROFESSOR_OAK, $517b, $5170
-	unknown_data_20000 $0c, ENERGY_SEARCH,          $51aa, $519a
-	unknown_data_20000 $03, POKEDEX,                $52dc, $52b4
-	unknown_data_20000 $07, FULL_HEAL,              $5428, $541d
-	unknown_data_20000 $0a, MR_FUJI,                $54a7, $5497
-	unknown_data_20000 $0a, SCOOP_UP,               $5506, $54f1
-	unknown_data_20000 $02, MAINTENANCE,            $562c, $560f
-	unknown_data_20000 $03, RECYCLE,                $56b8, $569a
-	unknown_data_20000 $0d, LASS,                   $5768, $5755
-	unknown_data_20000 $04, ITEM_FINDER,            $57b1, $578f
-	unknown_data_20000 $01, IMAKUNI_CARD,           $581e, $5813
-	unknown_data_20000 $01, GAMBLER,                $5875, $582d
-	unknown_data_20000 $05, REVIVE,                 $58a9, $5899
-	unknown_data_20000 $0d, POKEMON_FLUTE,          $58e8, $58d8
-	unknown_data_20000 $05, CLEFAIRY_DOLL,          $5982, $5977
-	unknown_data_20000 $05, MYSTERIOUS_FOSSIL,      $5982, $5977
-	unknown_data_20000 $02, POKE_BALL,              $59c6, $59a6
-	unknown_data_20000 $02, COMPUTER_SEARCH,        $5b34, $5b12
-	unknown_data_20000 $02, POKEMON_TRADER,         $5d8f, $5d7a
+	unknown_data_20000 $07, POTION,                 AIDecidePotion1,                        AIPlayPotion
```

Comment:
Might worth considering AIDecide_ and AIPlay_ prefixes (with the added underscore) foreasier distinction? Since there are a bunch of them that belong to the same package.

Reply:
Agreed.